projects/wireshark/build.sh

```diff
@@ -20,9 +20,41 @@
 FUZZ_DISSECTORS="ip \
   udp"
 
+FUZZ_TCP_PORT_DISSECTORS="bgp"
+FUZZ_UDP_PORT_DISSECTORS="dns"
+
+# generate_fuzzer <fuzzer_target> <fuzzer_cflags>
+generate_fuzzer()
+{
+  local fuzzer_target="$1" fuzzer_cflags="$2" fuzzer_name
+
+  fuzzer_name="fuzzshark_$1"
+
+  # -I$SRC/wireshark is correct, wireshark don't install header files.
+  $CC $CFLAGS -I $SRC/wireshark/ `pkg-config --cflags glib-2.0` \
+      $SRC/wireshark/tools/oss-fuzzshark.c \
+      -c -o $WORK/${fuzzer_name}.o \
+      $fuzzer_cflags
+
+  $CXX $CXXFLAGS $WORK/${fuzzer_name}.o \
+      -o $OUT/${fuzzer_name} \
+      ${WIRESHARK_FUZZERS_COMMON_FLAGS}
+
+  echo -en "[libfuzzer]\nmax_len = 1024\n" > $OUT/${fuzzer_name}.options
+  if [ -d "$SAMPLES_DIR/${fuzzer_target}" ]; then
+    zip -j $OUT/${fuzzer_name}_seed_corpus.zip $SAMPLES_DIR/${fuzzer_target}/*/*.bin
+  fi
+}
+
 export WIRESHARK_INSTALL_PATH="$WORK/install"
 mkdir -p "$WIRESHARK_INSTALL_PATH"
 
+# Prepare Samples directory
+SAMPLES_DIR="$WORK/samples"
+mkdir -p "$SAMPLES_DIR"
+git clone --depth=1 https://bitbucket.org/jwzawadzki/wireshark-fuzzdb.git $WORK/wireshark-fuzzdb/
```

Comment:
I have problem with putting it in the docker file. Cause of docker caching - it's much harder to test with new samples. (i.e. I need to invalidate docker cache, before docker run), and with samples being downloaded during build.sh, I don't need to :) Do you maybe know solution for it? Anyway, if required of course I can change.

btw. I've also generated samples for: bootp, bfd and bazaar, which could be include it in this changeset. But bfd dissector is adding only basic informations, using standard API, which should never crash. And Bazaar have known loop vulnerability, which is not yet fixed.
So it makes sense only to add bootp one?